### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NerevarineRule**
## Proposed changes

Added a sample ObjC code for querying multiple accounts

## Type of change

- [ ] Feature work
- [ ] Bug fix
- [x] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [x] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -379,6 +379,72 @@ parameters.completionBlockQueue = dispatch_get_main_queue();
 }];
```
Comment (kaisong1990) [inline README.md:379]: This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.

Comment (NerevarineRule) [reply README.md:379]: ?? is this a bot?

Comment (kaisong1990) [reply README.md:379]: Yes, feel free to resolve it if no need to update CHANGELOG.md

**Review by kaisong1990 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -379,6 +379,72 @@ parameters.completionBlockQueue = dispatch_get_main_queue();
...
+if (@available(iOS 13.0, *)) //Currently, this public API requires iOS version 13 or greater.   
+{
+    [application accountsFromDeviceForParameters:parameters
+                                 completionBlock:^(NSArray<MSALAccount *> * _Nullable accounts, __unused NSError * _Nullable error)
+    {
+        if (accounts)
+        {
+            for (MSALAccount *object in accounts)
+            {
+                if ([object.identifier isEqualToString:@"61cd1446-f9b4-403b-b137-1ffc975e0319.d34a1bb7-3481-4d5f-8b94-f3cc27bf8eac"])
```
Comment (oldalton) [inline README.md:422]: This isn't needed if you pass identifier property to MSALAccountEnumerationParameters

File: README.md [hunk 3]
```diff
@@ -379,6 +379,72 @@ parameters.completionBlockQueue = dispatch_get_main_queue();
 }];
 ```
 
+### Multiple Account Mode
+
+MSAL also provides a public API to query multiple accounts, granted that they exist in the MSAL cache. 
+1) First import MSALSilentTokenParameters.h in addition to the other core public libraries that are required to use the public APIs.
```
Comment (oldalton) [inline README.md:385]: I think developers should just import the umbrella header (MSAL/MSAL.h) and it should be part of the setup steps.

**Pull request closed without merging**